src/main/java/com/google/cloud/genomics/dataflow/utils/GCSHelper.java

```diff
@@ -0,0 +1,202 @@
+package com.google.cloud.genomics.dataflow.utils;
+
+import static com.google.api.services.storage.StorageScopes.DEVSTORAGE_READ_ONLY;
+
+import com.google.api.client.http.HttpHeaders;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.genomics.GenomicsScopes;
+import com.google.api.services.storage.Storage;
+import com.google.api.services.storage.StorageScopes;
+import com.google.api.services.storage.model.StorageObject;
+import com.google.cloud.genomics.utils.GenomicsFactory;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Lists;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.math.BigInteger;
+import java.security.GeneralSecurityException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.annotation.Nullable;
+
+/**
+ * A helper class to download parts of files.
```

Comment:
Agreed that there is some overlap but not enough that we wouldn't want this little helper.

I would like to keep this code in src since it is not a test. Yes, our first target use is for a test, but this can also be used in other places. In particular since GCD doesn't have a direct way for the client to get the result of the computation, I expect we may sometimes have a GCD job that ends with a GCD writer, and then the client loads whatever information it needs for display from GCS, using this code.